docsite/docs/fal-serverless/isolated_functions/serving.md

```diff
@@ -0,0 +1,67 @@
+---
+sidebar_position: 5
+---
+
+# Serving functions though a managed web server
+
+Serving an `@isolated` function exposes the function through a webserver managed by `fal-serverless`.
+
+Simplest way to serve a function is to mark the function you want to serve using the `@isolated` annotation's `serve=True` option and `fal-serverless` cli command.
```

Comment:
Should we stick to _decorator_ instead of _annotation_ when referring to `@isolated`?